### PR TITLE
Add ready property to DataSourceDisplay

### PR DESCRIPTION
**Pull request by hpinkos**
`dataSourceDisplay.ready` is true when everything in the data source collection is ready

@mramato There's no good way to do a `readyPromise` right?  Since things can get added to the `DataSourceCollection` whenever?

### CODE REVIEW

**Comment by mramato** [comment]
Yes, Promises are only good when you have a single answer that doesn't change.  In this case ready is going to change between true and false whenever static Entity data changes or data sources are added/removed.

File: Source/DataSources/DataSourceDisplay.js [hunk 1]
```diff
@@ -96,6 +96,8 @@ define([
         var defaultDataSource = new CustomDataSource();
         this._onDataSourceAdded(undefined, defaultDataSource);
         this._defaultDataSource = defaultDataSource;
+
+        this._ready = false;
```
Comment (mramato) [inline Source/DataSources/DataSourceDisplay.js]: Default to true, an empty DataSourceDisplay is always ready.

Comment (mramato) [inline Source/DataSources/DataSourceDisplay.js]: Actually, you're right.  Ignore me.

File: Specs/DataSources/DataSourceDisplaySpec.js [hunk 2]
```diff
@@ -280,6 +281,22 @@ defineSuite([
         expect(source2Visualizer.updatesCalled).toEqual(1);
     });
 
+    it('ready is true when datasources are ready', function() {
```
Comment (mramato) [inline Specs/DataSources/DataSourceDisplaySpec.js]: Change this test to check that both true and false get returned depending on the values returned by update (i.e. have one data source return true and then have it return false and check ready after both times).  You can spy on MockVisulizer and have it return whatever you want.

**Comment by mramato** [comment]
Just the one test comment.  Update CHANGES as well.  Thanks @hpinkos

**Comment by hpinkos** [comment]
@mramato ready

File: CHANGES.md [hunk 3]
```diff
@@ -29,6 +29,7 @@ Change Log
 * `Camera.flyTo` now honors max and min zoom settings in `ScreenSpaceCameraController`.
 * Fixed flying to `latitude, longitude, height` in the Geocoder.
 * Upgraded Knockout from version 3.2.0 to 3.4.0.
+* Added `DataSouceDisplay.ready`
```
Comment (mramato) [inline CHANGES.md]: You should explain what `ready` actually means in this case.  i.e. ready is true whenever any static data associated with the Entity API has been rendered and false when data is waiting to be rendered.

**Comment by hpinkos** [comment]
@mramato ready

**Pull request merged**